iceoryx_posh/source/roudi/roudi.cpp

```diff
@@ -335,8 +334,7 @@ void RouDi::processMessage(const runtime::MqMessage& message,
         }
         else
         {
-            capro::ServiceDescription service(
-                cxx::Serialization(cxx::CString100(cxx::TruncateToCapacity, message.getElementAtIndex(2))));
+            capro::ServiceDescription service(cxx::Serialization(std::string(message.getElementAtIndex(2))));
```

Comment:
to use CString100 was an error in the first place. Changed it back to std::string and made it a problem of the future elBoberido

Reply:
I leave this comment here to remind the future dkroenke that he reminds the future elBoberido about that problem ;-)

Reply:
This PR removes the old CStrin100 and not the std::string, therefore fine for now, right?

Reply:
yes, replacing std::string is the next topic on my todo list